solutions/slackbot/api/events.ts

```diff
@@ -5,24 +5,28 @@ export const config = {
   maxDuration: 30,
```

Comment:
Since the [maxDuration limit](https://vercel.com/docs/functions/runtimes#max-duration) for the hobby plan is 10s, I'm concerned about this config to bring misunderstandings among the hobby plan users.

Reply:
I'm okay to change to 10 👍